subsys/bluetooth/shell/bt.c

```diff
@@ -208,6 +285,10 @@ static void scan_recv(const struct bt_le_scan_recv_info *info,
 		return;
 	}
 
+	if (scan_verbose_output) {
+		/* call to bt_data_parse consumes netbufs so deep clone for verbose output */
+		net_buf_simple_init_with_data(buf_copy, buf->data, buf->len);
```

Comment:
Error in comment: This is a shallow clone. Also consider using `net_buf_simple_clone` instead.

Reply:
You are right. That made things a bit easier :)